tests/common/testPoint.cpp

```diff
@@ -248,10 +248,10 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(unscale_point, T, SignedTypes)
     BOOST_TEST(resultUnsigned == pt2 / UnsignedPoint::all(scale), epsilon % tolerance());
 
     pt /= scale;
-    BOOST_TEST(pt == result);
+    BOOST_TEST(pt == result, epsilon % tolerance());
 
     pt2 /= scale;
-    BOOST_TEST(pt2 == resultUnsigned);
+    BOOST_TEST(pt2 == resultUnsigned, epsilon % tolerance());
```

Comment:
i thought floating point test should be done via i.e `BOOST_TEST(x == y, tolerance(0.0005));` ? or is that not available in our boost versions?

Reply:
It seems `epsilon % tolerance()` is equivalent to `tolerance(epsilon/100)`. I think the first came up in a google search/boost docu first, so that's why I used it.